common/network/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 include_directories(${CMAKE_SOURCE_DIR}/common)
 
 add_library(network STATIC
+  UnixSocket.cxx
```

Comment:
Does any patch also make use of this? Having a bunch of dead code isn't really desirable.

Reply:
Not yeat. We have https://github.com/TigerVNC/tigervnc/issues/160 though, but I can exclude UnixSocket.cxx if preferred?

Reply:
I think so. It still has a lot of work left before it could be used in either the client or server.